unity-renderer/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -195,7 +198,8 @@ internal IEnumerator ProcessProfileWithEnabledCulling(CullingControllerProfile p
                     if (IsAvatarRenderer(mat))
                         shouldHaveShadow &= TestAvatarShadowRule(profile, distance);
 
-                    skr.updateWhenOffscreen = false;
+                    if (offScreenUpdate)
+                        skr.updateWhenOffscreen = false;
```

Comment:
this looks a bit strange (but maybe I miss details) - when `offScreenUpdate` is enabled, then disable `updateWhenOffscreen` on skinned meshes. Is it because then it handled by some our system?

Reply:
`offScreenUpdate` defines if we are updating `UpdateWhenOffscreen` or not. If you have a better naming I'm all for it!